dashboard/src/components/Drawer.js

```diff
@@ -83,79 +82,72 @@ class Drawer extends React.Component {
     this.props.router.push("/login");
   };
 
+  linkTo = path => {
+    const { router, match, onToggle } = this.props;
+    const fullPath = makeNamespacedPath(match.params)(path);
+    return () => {
+      router.push(fullPath);
+      onToggle();
+    };
+  };
+
   render() {
-    const { viewer, open, router, match, onToggle, classes } = this.props;
+    const { viewer, environment, open, onToggle, classes } = this.props;
     const { preferencesOpen } = this.state;
-    const linkTo = path => {
-      const fullPath = makeNamespacedPath(match.params)(path);
-      return () => {
-        router.push(fullPath);
-        onToggle();
-      };
-    };
 
     return (
       <MaterialDrawer variant="temporary" open={open} onClose={onToggle}>
         <div className={classes.paper}>
-          <div className={classes.header}>
-            <div className={classes.row}>
-              <IconButton
-                onClick={onToggle}
-                className={classes.hamburgerButton}
-              >
-                <MenuIcon />
-              </IconButton>
-              <img alt="sensu" src={logo} className={classes.logo} />
-            </div>
-            <div className={classes.row}>
-              {/* TODO update with global variables or whatever when we get them */}
-              <div className={classes.namespaceIcon}>
-                <OrganizationIcon
-                  icon="HalfHeart"
-                  iconColor="#FA8072"
-                  size={36}
+          <div className={classes.headerContainer}>
+            <div className={classes.header}>
+              <div className={classes.row}>
+                <IconButton
+                  onClick={onToggle}
+                  className={classes.hamburgerButton}
+                >
+                  <MenuIcon />
+                </IconButton>
+                <Wordmark
+                  alt="sensu"
+                  className={classes.logo}
+                  color="secondary"
+                />
+              </div>
+              <div className={classes.row}>
+                {/* TODO update with global variables or whatever when we get them */}
+                <div className={classes.namespaceIcon}>
+                  <EnvironmentIcon environment={environment} size={36} />
```

Comment:
I don't think the comment on like 117 is needed any more because it looks like we have a variable for environment